controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1707,30 +1707,35 @@ func (r *BareMetalHostReconciler) updateEventHandler(e event.UpdateEvent) bool {
 }
 
 // SetupWithManager registers the reconciler to be run by the manager
-func (r *BareMetalHostReconciler) SetupWithManager(mgr ctrl.Manager, preprovImgEnable bool) error {
+func (r *BareMetalHostReconciler) SetupWithManager(mgr ctrl.Manager, preprovImgEnable bool, options controller.Options) error {
```

Comment:
It's weird that an Options structure is passed in, but all of the options except MaxConcurrentReconciles are ignored.

Reply:
Good point, fixed.